doc/openocd.texi

```diff
@@ -10716,6 +10716,28 @@ Perform a 32-bit DMI read at address, returning the value.
 Perform a 32-bit DMI write of value at address.
 @end deffn
 
+@subsection RISC-V Trigger Commands
+
+The RISC-V Debug Specification defines several trigger types that don't map
+cleanly onto OpenOCD's notion of hardware breakpoints. These commands let you
+set those triggers directly. (It's also possible to do so by writing the
+appropriate CSRs.)
+
+@deffn {Command} {riscv itrigger set} [@option{m}] [@option{s}] [@option{u}] [@option{vs}] [@option{vu}] [@option{nmi}] mie_bits
+Set a type 4 trigger on the current target. @option{m}, @option{s}, @option{u},
+@option{vs}, and @option{vu} control which execution modes the trigger fires in.
+If [@option{nmi}] is passed then the trigger will on fire non-maskable
+interrupts in those modes.  mie_bits controls which interrupts the trigger fires
+on, using the same bit assignments as in the mie CSR (defined in the RISC-V
```

Comment:
Minor: Is there any suitable formatting that could be used for "mie" in this sentence?

Reply:
Maybe, but I can't easily find it. Browsing through the document, I see e.g.
```
@deffn {Command} {nrf5 info}
Decodes and shows information from FICR and UICR registers.
@end deffn
```
which doesn't use any special formatting.